mininterface/_tk_interface/utils.py

```diff
@@ -67,12 +69,55 @@ def get(self):
 
 
 def choose_file_handler(variable: Variable, tag: PathTag):
+    """Handler for file/directory selection on PathTag"""
     def _(*_):
-        if tag.multiple:
-            out = str(list(askopenfilenames(title="Choose files")))
+        initialdir = tag.val if tag.val else os.getcwd()
```

Comment:
you are guaranteed the tag.val is Path or their sequence. If there are multiple of them, take the first or something. Now, if they are multiple, you end up in cwd.

And cwd is the default behaviour, you don't have to set it from os explicitly I think.